components/lib/VDropdown.vue

```diff
@@ -9,8 +9,8 @@
     <div class="custom-select__trigger font-serif color-primary cursor-pointer">
       <span v-if="value">{{ value }}</span>
       <span v-else class="placeholder">{{ placeholder }}</span>
-      <div class="arrow">
-        <v-image path="icons/arrow-down.svg" />
+      <div class="arrow" @click.stop="toggle">
+        <v-image path="icons/arrow-down.svg" @click.stop="toggle" />
```

Comment:
For some reason these clicking on area where these elements are present weren't triggering dropdown's click event so added these click events. Will need to investigate why it was happening